src/protocol.h

```diff
@@ -67,7 +67,13 @@ class CMessageHeader
 
 /** nServices flags */
 enum {
+    // NODE_NETWORK means that the node is capable of serving the block chain. It is currently
+    // set by all Bitcoin Core nodes, and is unset by SPV clients or other peers that just want
+    // network services but don't provide them.
     NODE_NETWORK = (1 << 0),
+    // NODE_GETUTXO means the node is capable of responding to the getutxo protocol request.
+    // Bitcoin Core does not support this but a patch set called Bitcoin XT does.
+    NODE_GETUTXO = (1 << 1),
```

Comment:
Yes, the BIP should be mentioned here. That's more important than advertising what software implements it.